Sources/JavaScriptKit/ConvertibleToJSValue.swift

```diff
@@ -58,7 +58,11 @@ extension UInt32: ConvertibleToJSValue {
 }
 
 extension UInt64: ConvertibleToJSValue {
+#if JAVASCRIPTKIT_WITHOUT_BIGINTS
```

Comment:
I think it's reasonable to remove `ConvertibleToJSValue` conformance from `UInt64` when no BigInt support.
Users can cast it to a smaller type at their own risk.